packages/ckeditor5-ui/src/panel/balloon/balloonpanelview.ts

```diff
@@ -194,6 +203,14 @@ export default class BalloonPanelView extends View {
 		} );
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	public override destroy(): void {
+		this.hide();
```

Comment:
Why only `hide()`? Why not `unpin()`?

Reply:
@niegowski These two methods do the same, most of the time. I think that `hide` is a little bit safer. `unpin` does something like this:

```tsx
	public unpin(): void {
		if ( this._pinWhenIsVisibleCallback ) {
            ...
``` 

So `hide` might (in theory) keep `isVisible = true` when `_pinWhenIsVisibleCallback`  is `null`. Is this real case? Don't know.